infra/cifuzz/cifuzz.py

```diff
@@ -190,6 +195,24 @@ def build_fuzzers(self):
       return False
     return True
 
+  def handle_msan(self):
+    """Copies MSAN libs to |msan_libs_dir| and returns docker arguments to use
+    that directory for MSAN libs."""
+    logging.info('Copying MSAN libs.')
+    os.makedirs(self.MSAN_LIBS_PATH, exist_ok=True)
+    msan_volume_arg = '{msan_libs_path}:{msan_libs_path}'.format(
```

Comment:
nit: nit: f-style var instead of format everywhere.

Reply:
Don't think I can do this. this isn't run using a recent enough python3.

Reply:
fine then, ignore all of these.